ip_map_branch.txt

```diff
@@ -15,7 +15,7 @@ three	https://github.com/openemr/openemr.git	rel-703	0	0	1	0	0	0	1	three.openemr
 three_a	https://github.com/openemr/openemr.git	rel-703	0	0	0	0	demo_5_0_0_5.sql	0	1	three.openemr.io/a	hey	branch	5.0.0	0	0	0	Public OpenEMR 7.0.3 Development Demo With Demo Data on Ubuntu 24.04 (with nodejs 20)
 four	https://github.com/stephenwaite/openemr.git	rel-703-p5a	0	0	0	0	0	0	1	four.openemr.io	hey	branch	0	0	0	0	UP FOR GRABS Public OpenEMR Demo - Alpha
 four_a	https://github.com/ophthal/openemr.git	GFS8	0	0	0	0	demo_5_0_0_5.sql	0	1	four.openemr.io/a	hey	branch	5.0.0	0	0	0	UP FOR GRABS Public OpenEMR Demo - Beta With Demo Data
-four_b	https://github.com/openemr/openemr.git	master	0	0	0	0	0	0	1	four.openemr.io/b	hey	branch	0	0	0	0	UP FOR GRABS Public OpenEMR Demo - Gamma
+four_b	https://github.com/psoas/openemr.git	patient-finder-update-2	0	0	0	0	demo_5_0_0_5.sql	0	1	four.openemr.io/b	hey	branch	0	0	0	0	UP FOR GRABS Public OpenEMR Demo - Patient Finder Update  With Demo Data
```

Comment:
hi @psoas , super close. since using demo data, just need a 5.0.0 in the column to right of the one that has `branch`. this tells the demo to start at version 5.0.0 on the auto upgrade (since the demo data is from version 5.0.0)

Reply:
correction made and committed